### DISCUSSION TITLE
1.5.1下6云原生阶段，如图 1-5 所示 应改为  如图 1-18 所示

### DISCUSSION THREAD

**Discussion by volcanG** [isno/theByteBook#122] [General]
1.5.1 下 6云原生阶段，如图 1-5 所示 应改为  如图 1-18 所示
[
![无标11题](https://github.com/isno/theByteBook/assets/18587190/f53e387e-8ef6-44d9-8197-c420b9bd5dee)
](url)

**Comment by isno** [comment]
已修改，感谢。
